kustomize/istio-telemetry/mixer-telemetry.yaml

```diff
@@ -486,7 +486,7 @@ spec:
           - unix:///sock/mixer.socket
           - --log_output_level=default:info
           - --configStoreURL=mcp://localhost:15019
-          - --configDefaultNamespace=istio-control
```

Comment:
I thought this was the namespace of galley? is this where the configs like CRDs are stored?

Reply:
This is the "root" namespace as seen by mixer. for telemetry related stuff, we do not need it to be istio-control.